xml/esxi-performace.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<config>
+    <global>
+        <logfileRootDir>TestResults</logfileRootDir>
+        <defaultSnapshot>ICABase</defaultSnapshot>
+        <email>
+            <recipients>
+                <to>myboss@mycompany.com</to>
+                <to>myself@mycompany.com</to>
+            </recipients>
+            <sender>myself@mycompany.com</sender>
+            <subject>ESX demo Test</subject>
+            <smtpServer>mysmtphost.mycompany.com</smtpServer>
+        </email>
+    </global>
+
+    <testSuites>
+        <suite>
+            <suiteName>debug_demo_suite</suiteName>
+            <suiteTests>
+                <suiteTest>perf_stor_iozone_stress</suiteTest>
+                <suiteTest>perf_stor_fio_SCSI_RAW</suiteTest>
+                <suiteTest>perf_stor_fio_SCSI_FS</suiteTest>
+                <suiteTest>perf_stor_fio_IDE_FS</suiteTest>
+                <suiteTest>perf_stor_fio_IDE_RAW</suiteTest>
+                <suiteTest>perf_stor_fio_NVMe_FS</suiteTest>
+                <suiteTest>perf_stor_fio_NVMe_RAW</suiteTest>
+            </suiteTests>
+        </suite>
+    </testSuites>
+
+    <testCases>
+        <test>
+            <testName>perf_stor_fio_NVMe_FS</testName>
+            <testID>ESX-PERF-010</testID>
```

Comment:
1. Line 35, this is the 10th cases? this number is correct? please double check it